grpc/execution/server.go

```diff
@@ -183,7 +101,7 @@ func (s *ExecutionServiceServerV1) GetBlock(ctx context.Context, req *astriaPb.G
 	res, err := s.getBlockFromIdentifier(req.GetIdentifier())
 	if err != nil {
 		log.Error("failed finding block", err)
-		return nil, err
+		return nil, shared.WrapError(err, "failed finding block")
```

Comment:
In general I don't think we want to wrap these errors, they are specifically gRPC error codes and are a part of the API vs something internal for tracing.

Reply:
Addressed in https://github.com/astriaorg/flame/pull/30/commits/23a2bd7a275ea8628fc6d53b93665dbe117ac8ce